docs/first_steps/first_steps.rst

```diff
@@ -52,6 +52,11 @@ There are multiple parameters that the user can define in this menu:
   The latter will send that last stored values to subscribers joining after the publisher has been created. (see
   `Fast DDS DurabilityQosPolicy Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/standardQosPolicies.html#durabilityqospolicy>`_).
 
+- **Data representation:** The Data representation version is set to version 1,
+  ``XCDR``, but the user can select the representation version 2, ``XCDR2``.
+  (see
+  `Fast DDS Data representation Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/api_reference/dds_pim/core/policy/datarepresentationid.html>`_).
```

Comment:
It would be better redirecting to the explanatory section instead of the `api-reference` 
https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/xtypesExtensions.html#datarepresentationqospolicy